lib/send-mail.ts

```diff
@@ -115,7 +115,7 @@ export function parseMBoxMessageIDAndReferences(parsed: IParsedMBox):
         /^\s*<([^>]+)>(\s*|,)(\([^")]*("[^"]*")?\)\s*|\([^)]*\)$)?(<.*)?$/;
     for (const header of parsed.headers ?? []) {
         if (header.key === "In-Reply-To" || header.key === "References") {
-            let value: string = header.value;
+            let value: string = header.value.replace(/[\r\n]/g, " ");
```

Comment:
Should it be `\r?\n` in case this is called in the future with an mbox that has already removed all `\r`?

Reply:
I'd rather treat even stray `\r` as whitespace.

Reply:
Sorry for being so curt... it was a little late for me and I wanted to go to bed :smile:

Reply:
> I wanted to go to bed

No problem.  Any time you post during my day I pay attention.

Speaking of going to bed...